mscore/jackaudio.cpp

```diff
@@ -33,6 +33,7 @@
 #include "seq.h"
 #include "libmscore/score.h"
 #include "libmscore/repeatlist.h"
+#include "libmscore/sig.h"
```

Comment:
Why is this needed?

Reply:
Well, perhaps all these include changes are really better not to mix with this PR. I'll remove it now from here.

Reply:
the removals looks OK to me though